server/datastore/mysql/migrations/tables/20251222174712_AddHostLocationSupport.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20251222174712, Down_20251222174712)
+}
+
+func Up_20251222174712(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+	CREATE TABLE host_locations (
+		host_id INT UNSIGNED NOT NULL,
+		latitude DECIMAL(10, 8),
+		longitude DECIMAL(11, 8),
+		created_at timestamp(6) DEFAULT CURRENT_TIMESTAMP(6),
+		updated_at timestamp(6) DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+
+		PRIMARY KEY (host_id),
+		FOREIGN KEY (host_id) REFERENCES hosts (id) ON DELETE CASCADE
```

Comment:
I think we intentionally don't do fkeys on `hosts` for performance reasons (we only have two in the entire schema per the check i did just now), so we want to drop this and add this table entry to the manual host cleanup on delete.

Reply:
yeah, there's even a nice CI job that checks 🤦